collie/model/base/base_pipeline.py

```diff
@@ -823,8 +815,13 @@ def user_user_similarity(self, user_id: int) -> pd.Series:
         user_embeddings = self._get_user_embeddings()
         user_embeddings = user_embeddings / user_embeddings.norm(dim=1)[:, None]
 
+        sim_score_idxs_series = self._similarity(embeddings=user_embeddings, id=user_id)
+
+        return sim_score_idxs_series
+
+    def _similarity(self, embeddings, id: int) -> pd.Series:
```

Comment:
Also, can we rename this to something like `_calculate_embedding_similarity` just to be a bit more descriptive of the helper method? What do you think?

Reply:
perfect! thanks for the suggestion